src/librustc/mir/mod.rs

```diff
@@ -1188,17 +1174,6 @@ impl<'tcx> TerminatorKind<'tcx> {
     }
 }
 
-#[derive(Clone, Debug, RustcEncodable, RustcDecodable)]
-pub enum AssertMessage<'tcx> {
```

Comment:
Hmm, can you put `Box` around the use of `AssertMessage` in `TerminatorKind::Assert`? Just to avoid the memory usage blowup.